logging.tf

```diff
@@ -0,0 +1,85 @@
+# Base infra logging
+locals {
+  fluent_operator_namespace              = "fluent" # should we put it as a fluent_namespace variable ?
+  fluent_operator_helm_chart_version     = "v3.0.0" # variable ? and if "v3.0", then every terraform apply will update, even if there is no change
```

Comment:
put it directly in version field since we'd only change this will the module not to break anything.